authorization/README.md

```diff
@@ -6,4 +6,10 @@
 > [!NOTE]  
 > Adding your username to one of the `Group`s managed by this ArgoCD application does **not** create your username or grant access to the cluster. You must already have an internal SSO login in order for these role bindings to take effect.
 
-Membership in the `redhat-ai-dev-users` `Group` on the cluster grants cluster-wide `edit` permissions. Additional permissions can be configured via additional `Group` and `RoleBinding` resources.
\ No newline at end of file
+Membership in under the listed `Group` entities provides the corresponding permissions: 
+
+- `redhat-ai-dev-view-users` - grants cluster-wide `view` permissions
+- `redhat-ai-dev-edit-users` - grants cluster-wide `edit` permissions
+- `rhoai-workspace-edit-users` - grants `edit` permissions to the `rhoai-workspace` namespace 
```

Comment:
Instead of having a `rhoai-workspace-edit-users`, let's have a `RoleBinding` which binds the `redhat-ai-dev-users` group to the `edit` cluster role. This will consolidate cluster access and RHOAI access under a single Group which should be sufficient for our use cases